README.md

```diff
@@ -34,7 +34,7 @@ Data types by Apache Calcite: https://calcite.apache.org/docs/reference.html#dat
 ## Configuration
 
 - **query**: SQL to run (string)
-- **default_timezone**: If the sql type of a column is `date`/`time`/`datetime` and the embulk type is `string`, column values are formatted int this default_timezone. You can overwrite timezone for each columns using column_options option. (string, default: `UTC`)
+- **default_timezone**: Set JDBC connect 'timeZone' param (string)
```

Comment:
Hmm, I don't understand what should be specified to this configuration from the description. Can we have:

1. Required or optional
2. How to represent the timezone to specify (link to any standard spec, or JDBC documentation?)
3. Which parts and how does this configuration impact?